app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/item/VaultItemViewModel.kt

```diff
@@ -758,9 +758,19 @@ class VaultItemViewModel @Inject constructor(
 
     private fun handleSshKeyTypeActions(action: VaultItemAction.ItemType.SshKey) {
         when (action) {
+            VaultItemAction.ItemType.SshKey.CopyPublicKeyClick -> handleCopyPublicKeyClick()
             is VaultItemAction.ItemType.SshKey.PrivateKeyVisibilityClicked -> {
                 handlePrivateKeyVisibilityClicked(action)
             }
+
+            VaultItemAction.ItemType.SshKey.CopyFingerprintClick -> handleCopyFingerprintClick()
+        }
+    }
+
+    private fun handleCopyPublicKeyClick() {
+        onSshKeyContent { content, sshKey ->
+            val publicKey = requireNotNull(sshKey.publicKey)
```

Comment:
We won't need this requireNotNull after the rebase, right?

Reply:
Ha. Yup. You're too fast. 😆